src/PowerShellEditorServices/Services/DebugAdapter/Handlers/LaunchAndAttachHandler.cs

```diff
@@ -237,6 +241,12 @@ public async Task<AttachResponse> Handle(PsesAttachRequestArguments request, Can
 
         private async Task<AttachResponse> HandleImpl(PsesAttachRequestArguments request, CancellationToken cancellationToken)
         {
+            cancellationToken.Register(() =>
+                {
+                    if (Runspace.DefaultRunspace != null)
+                    {
+                    }
+                });
```

Comment:
What's this do?

Reply:
Removed! That was there as a debugging thing and I was adding `Debug.WriteLine` or something as a test.